build-system/server/app.js

```diff
@@ -755,6 +761,14 @@ app.use('/impression-proxy/', (req, res) => {
   // Or fake response with status 204 if viewer replaceUrl is provided
 });
 
+app.get('/run-variable-substitution', runVariableSubstitution);
+
+app.get('/save-request', saveRequest);
+
+app.get('/save-variables', saveVariables);
+
+app.get('/get-request', getRequest);
```

Comment:
Add a comment at the top of these functions explaining their general purpose (and usage, where applicable).

Reply:
I suggest rename this to something that's more obviously linked to variable substitution. Something like `/save-variable-request`?

Reply:
Good idea, done